src/app/common/search-input/search-input.component.ts

```diff
@@ -1,14 +1,13 @@
-import { Component, OnInit, Input, Output, EventEmitter } from '@angular/core';
+import { Component, Input, Output, EventEmitter } from '@angular/core';
 
 @Component({
 	selector: 'asy-search-input',
 	templateUrl: './search-input.component.html',
 	styleUrls: ['./search-input.component.scss']
 })
-export class SearchInputComponent implements OnInit {
+export class SearchInputComponent {
 
 	@Input() placeholder: string = 'Search...';
-	@Input() width: string = '500';
 	@Output() applySearch: EventEmitter<string> = new EventEmitter();
 	search: string = '';
```

Comment:
Can we also add the ability to set default search? Like making search an Input, or having another input